frontend/src/fixtures/commonsFixtures.js

```diff
@@ -11,7 +11,9 @@ const commonsFixtures = {
             "milkPrice": 10,
             "degradationRate": .5,
             "showLeaderboard": true,
+            "capacityPerUser": 10,
             "carryingCapacity": 100,
+            "effectiveCapacity": 100,
```

Comment:
If the totalPlayers are 50, and the capacityPerUser is 10, shouldn't the effectiveCapacity be 500?